### PR TITLE
WIP: improving readability and adding some more comments

### PR DESCRIPTION
**Pull request by GregorySech**
I'm doing another PR since I'll be adding some more documentation today.
I've found some things that I am not sure if should be removed or refactored.
While I was at it I've tried to improve readability using the new list creation statements.

### CODE REVIEW

**Comment by johnpryan** [comment]
Are more changes incoming?

**Comment by GregorySech** [comment]
yeah, I've added a couple of todos on things that I've changed but I was not sure about.
Same for a piece of math that didn't add up

**Comment by GregorySech** [comment]
Whelp static analysis didn't pick that up.

File: lib/src/layer/polygon_layer.dart [hunk 1]
```diff
@@ -41,6 +41,7 @@ class PolygonLayer extends StatelessWidget {
   Widget build(BuildContext context) {
     return LayoutBuilder(
       builder: (BuildContext context, BoxConstraints bc) {
+        // TODO unused BoxContraints should remove?
```
Comment (escamoteur) [inline lib/src/layer/polygon_layer.dart]: This one isn't useless its used the line below.

**Pull request merged**